src/main/java/com/csselect/database/mysql/MysqlDatabaseAdapter.java

```diff
@@ -0,0 +1,424 @@
+            if (set.next()) {
+                return null;
+            }
+            set.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
+        MysqlPlayerAdapter adapter;
+        try {
+            adapter = new MysqlPlayerAdapter();
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
+        adapter.setEmail(email);
+        adapter.setPassword(hash, salt);
+        adapter.setUsername(username);
+        return new Player(adapter);
+    }
+
+    @Override
+    public Organiser createOrganiser(String email, String hash, String salt) {
+        try {
+            ResultSet set = executeMysqlQuery("SELECT * FROM organisers WHERE (email='" + email + "');");
+            if (set.next()) {
+                return null;
+            }
+            set.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
+        MysqlOrganiserAdapter adapter;
+        try {
+            adapter = new MysqlOrganiserAdapter();
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
+        adapter.setEmail(email);
+        adapter.setPassword(hash, salt);
+        return new Organiser(adapter);
+    }
+
+    @Override
+    public void registerGame(Organiser organiser, Game game) {
+        if (!gameMap.containsKey(game)) {
+            gameMap.put(game, organiser);
+            try {
+                executeMysqlUpdate("UPDATE organiser_id SET " + organiser.getId() + " FROM games WHERE"
+                        + " (id='" + game.getId() + "');");
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    @Override
+    public void removeGame(Game game) {
+        gameMap.remove(game);
+        try {
+            executeMysqlUpdate("DELETE FROM games WHERE (id='" + game.getId() + "');");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Executes the given Mysql-query on the main database
+     * @param query query to execute
+     * @return ResultSet of the operation
+     * @throws SQLException Thrown when there is an error executing the given statement
+     */
+    ResultSet executeMysqlQuery(@Language("sql") String query) throws SQLException {
+        return executeMysqlQuery(query, PRODUCT_DATABASE_NAME);
+    }
+
+    /**
+     * Executes the given Mysql-query on the given database
+     * @param query query to execute
+     * @param databaseName database to execute the query on
+     * @return ResultSet of the operation
+     * @throws SQLException Thrown when there is an error executing the given statement
+     */
+    ResultSet executeMysqlQuery(@Language("sql") String query, String databaseName) throws SQLException {
```

Comment:
maybe use prepared statments instead of just passing user input strings as sql commands